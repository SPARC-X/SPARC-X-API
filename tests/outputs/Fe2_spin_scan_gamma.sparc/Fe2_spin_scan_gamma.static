***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Fe:
      0.0000000000       0.0000000000       0.0000000000
      0.5200000000       0.5100000000       0.4900000000
Total free energy (Ha): -2.283157353113279E+02
Atomic forces (Ha/Bohr):
  8.0738249305E-01   3.7399117306E-01  -3.5796157735E-01
 -8.0738249305E-01  -3.7399117306E-01   3.5796157735E-01
Stress (GPa): 
 -2.1918863425E+04   1.3932450782E+03  -5.1023512490E+01 
  1.3932450782E+03  -2.1975897437E+04  -1.2676410947E+02 
 -5.1023512490E+01  -1.2676410947E+02  -2.2380745784E+04
