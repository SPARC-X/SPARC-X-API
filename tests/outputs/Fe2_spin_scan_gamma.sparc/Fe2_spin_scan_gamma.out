***************************************************************************
*                       SPARC (version Feb 03, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun Feb  5 13:41:30 2023                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 2.840052 2.840052 2.840052 
LATVEC:
1 0.1 0 
0.1 1 0 
0 0 1 
FD_GRID: 29 29 29
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 1
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001000000336
EXCHANGE_CORRELATION: SCAN
NSTATES: 24
CHEB_DEGREE: 51
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 1
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-05
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: kerker
MIXING_PRECOND_MAG: none
TOL_PRECOND: 9.65E-06
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_PARAMETER_MAG: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: Fe2_spin_scan_gamma/temp_run/Fe2_spin_scan_gamma
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
2.840052000000000 0.284005200000000 0.000000000000000 
0.284005200000000 2.840052000000000 0.000000000000000 
0.000000000000000 0.000000000000000 2.840052000000000 
Volume: 2.2678486634E+01 (Bohr^3)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 2
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 24
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 3 4 4
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  48
Mesh spacing in x-direction        :  0.0984213 (Bohr)
Mesh spacing in y-direction        :  0.0984213 (Bohr)
Mesh spacing in z-direction        :  0.0979328 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Fe2_spin_scan_gamma/temp_run/Fe2_spin_scan_gamma.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Fe 16
Pseudopotential                    :  Fe2_spin_scan_gamma/Fe_LDA.psp8
Pseudocharge radii of atom type 1  :  6.50 6.50 6.46 (x, y, z dir)
Number of atoms of type 1          :  2
Estimated total memory usage       :  63.26 MB
Estimated memory per processor     :  1.32 MB
========================================================================================
                    Self Consistent Field (SCF#1)                     
========================================================================================
Iteration     Free Energy (Ha/atom)    Magnetization     SCF Error        Timing (sec)
1            -1.1571204907E+02         2.5805E-08        5.433E-01        1.007
2            -1.1416113612E+02         2.3585E-08        3.472E-01        0.336
3            -1.1415213872E+02         1.5099E-08        1.191E-01        0.322
4            -1.1415478645E+02         3.7033E-09        1.291E-02        0.316
5            -1.1415757261E+02         1.1676E-09        3.021E-03        0.315
6            -1.1415782672E+02         4.8199E-10        1.104E-03        0.314
7            -1.1415785978E+02         1.2276E-10        1.723E-04        0.327
8            -1.1415786767E+02         8.0249E-11        9.984E-05        0.310
9            -1.1415786724E+02         2.2609E-11        2.129E-05        0.310
10           -1.1415786766E+02         1.0761E-11        7.565E-06        0.306
Total number of SCF: 10    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -1.1415786766E+02 (Ha/atom)
Total free energy                  : -2.2831573531E+02 (Ha)
Band structure energy              :  2.1159822900E+00 (Ha)
Exchange correlation energy        : -3.1398406932E+01 (Ha)
Self and correction energy         : -2.4173788380E+02 (Ha)
-Entropy*kb*T                      : -4.9458384163E-10 (Ha)
Fermi level                        :  2.1076013118E+00 (Ha)
RMS force                          :  9.5909977503E-01 (Ha/Bohr)
Maximum force                      :  9.5909977503E-01 (Ha/Bohr)
Time for force calculation         :  0.049 (sec)
Pressure                           :  2.2091835549E+04 (GPa)
Maximum stress                     :  2.2380745784E+04 (GPa)
Time for stress calculation        :  0.276 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  5.145 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
